.github/workflows/ci.yml

```diff
@@ -301,122 +243,132 @@ jobs:
+              run: |
+                  git clone https://github.com/spryker/robotframework-suite-tests.git --single-branch --branch master robotframework-tests
+            - name: Install Project
+              continue-on-error: true
               run: |
                   git clone https://github.com/spryker/docker-sdk.git ./docker
-                  docker/sdk boot deploy.ci.functional.mariadb.yml -v
+                  docker/sdk boot -v deploy.ci.acceptance.mariadb.dynamic-store.yml
+                  sudo bash -c "echo '127.0.0.1 backend-api.eu.spryker.local backend-api.us.spryker.local backend-gateway.eu.spryker.local backend-gateway.us.spryker.local backoffice.eu.spryker.local backoffice.us.spryker.local glue-backend.eu.spryker.local glue-backend.us.spryker.local glue-storefront.eu.spryker.local glue-storefront.us.spryker.local glue.eu.spryker.local glue.us.spryker.local mail.spryker.local mp.eu.spryker.local mp.us.spryker.local queue.spryker.local scheduler.spryker.local spryker.local swagger.spryker.local yves.eu.spryker.local yves.us.spryker.local' >> /etc/hosts"
                   docker/sdk up -t -v
-                  docker/sdk testing codecept run -c codeception.functional.yml
-
+            -   name: Custom commands
+                continue-on-error: false
+                run: |
+                    docker/sdk cli composer dump-autoload -o -a --apcu
+                    SPRYKER_CURRENT_REGION=EU docker/sdk console queue:worker:start --stop-when-empty
+            -   name: Run Tests
+                run: |
+                    cd robotframework-tests
+                    robot -v env:api_suite -v dms:true -v glue_env:http://glue.eu.spryker.local -v bapi_env:http://glue-backend.eu.spryker.local -v sapi_env:http://glue-storefront.eu.spryker.local -v ignore_console:false -d results --exclude skip-due-to-issueORskip-due-to-refactoring -s '*'.tests.api.suite.'*'.positive .
+                    touch results/time.txt && echo $(date) > results/time.txt
+            -   name: Upload artifacts
+                if: failure()
+                run: |
+                    AWS_DEFAULT_REGION=${{env.ROBOT_TESTS_ARTIFACTS_BUCKET_REGION}} AWS_ACCESS_KEY_ID=${{ secrets.ROBOT_TESTS_ARTIFACTS_KEY }} AWS_SECRET_ACCESS_KEY=${{ secrets.ROBOT_TESTS_ARTIFACTS_SECRET }} aws s3 cp robotframework-tests/results s3://${{vars.ROBOT_TESTS_ARTIFACTS_BUCKET}}/suite-dynamic-multistore/${GITHUB_RUN_ID}/PHP8.2MariaDBRobot/ \
+                    --recursive \
+                    --expires "$(date -d '+7 days' --utc +'%Y-%m-%dT%H:%M:%SZ')"
 
-    php-81-postgres-functional-debian:
-        name: "PHP 8.1 / PostgreSQL / Functional / Debian"
+    negative-docker-alpine-php-8-1-mariadb-robot-dynamic-multistore:
+        name: "[Hackathon] Negative / Docker / Alpine / PHP 8.1 / MariaDB / Robot / API / Dynamic Multistore"
         runs-on: ubuntu-20.04
+        if: "contains(github.event.pull_request.labels.*.name, 'hackathon')"
```

Comment:
Can we define global var for the `hackathon` label and re-use it in each pipeline?